db/db.json

```diff
@@ -1,52 +1,2502 @@
 {
-  "toys": [
+  "pokemon": [
     {
       "id": 1,
-      "name": "Woody",
-      "image": "http://www.pngmart.com/files/3/Toy-Story-Woody-PNG-Photos.png",
-      "likes": 8
+      "num": "001",
```

Comment:
As much as I love seeing Pokemon projects, could we just do data like
```json
{
  "id": 1,
  "exampleAttribute": "This is just an example. Please supply your own data."
}
```

Reply:
The only reason I didn't do that is because I wanted to include sample data that included a picture.

So maybe we can do something like that but with a boring image like this one 

`https://placehold.co/300x200.jpg`